swagger.yaml

```diff
@@ -10,25 +12,37 @@ basePath: "/v1"
 schemes:
   - http
 tags:
-  - name: "hello"
+  - name: "feedback"
   - name: "private"
+securityDefinitions:
+  InternalAPIKey:
+    name: internal-token
+    description: "API key used to allow only internal services to post feedback"
+    in: header
+    type: apiKey
 paths:
-  /hello:
-    get:
+  /feedback:
+    post:
       tags:
-        - hello
-      summary: Example hello world endpoint
-      description: Returns an example payload.
-      produces:
-        - application/json
+        - feedback
+      summary: "Post feedback for distribution"
+      description: "Post feedback forms here for distribution and/or storage internally"
+      requestBody:
```

Comment:
I checked the swagger spec against go-swagger and https://editor.swagger.io/ and both complained about the `requestBody`.
- go-swagger: `paths./feedback.post.requestBody in body is a forbidden property`
- swagger editor:
```
Structural error at paths./feedback.post
should NOT have additional properties
additionalProperty: requestBody
```

 I think this corresponds to openapi 3.0? In other repos we added lists of paramters and specified that they are in body. For example:

Specify a paramter (with ref): https://github.com/ONSdigital/dp-dataset-api/blob/develop/swagger.yaml#L253

Description of paramter, including `in:Body` (referenced by previous):
https://github.com/ONSdigital/dp-dataset-api/blob/develop/swagger.yaml#L94

Maybe something similar can be done in this repo.